cmd/chainbridge/deploy.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"encoding/hex"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/ChainSafe/ChainBridgeV2/keystore"
+	log "github.com/ChainSafe/log15"
+	"github.com/urfave/cli"
+)
+
+var deployCommand = cli.Command{
+	Action:   handleDeployCmd,
+	Name:     "deploy",
+	Usage:    "deploys test ganache",
+	Category: "TESTING",
+	Flags:    DeployFlags,
+	Description: "\tThe deploy command is used to deploy a ganache instance for testing\n" +
+		"\tTo add additional accounts, -testaccount\n" +
+		"\tTo only use custom accounts, -reset\n" +
+		"\tTo use a mnemonic, -mnemonic",
+}
+
+// The Private Keys of Alice, Bob, Etc
+var BaseAccounts = []string{
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.AliceKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.BobKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.CharlieKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.DaveKey].Encode()),
+	"0x" + hex.EncodeToString(keystore.TestKeyRing.EthereumKeys[keystore.EveKey].Encode()),
```

Comment:
`encode` serializes the entire keypair, it is not the private key.

I think this might be best solved by updating `keystore.TestKeyRingHolder` to hold maps of specific key types, please see #201

Reply:
I have merged #201 in, and it seems like `Encode` in the `secp256k1` library specifically only encodes the private key.